clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/DcosConfigurationProperties.groovy

```diff
@@ -40,4 +51,25 @@ class DcosConfigurationProperties {
   static class LinkedDockerRegistryConfiguration {
     String accountName
   }
+
+  public static Config buildConfig(final Account account, final Cluster cluster, final ClusterCredentials clusterConfig) {
```

Comment:
This was a weird issue, I moved these because for whatever reason when running this it would complain it couldn't find the new signature.